examples/rendering/interactive_flex.dart

```diff
@@ -37,25 +38,32 @@ class RenderImageGrow extends RenderImage {
 
 RenderImageGrow image;
 
-void handlePopRoute() {
-  activity.finishCurrentActivity();
-}
-
-final Map<int, Touch> touches = <int, Touch>{};
+class DemoBinding extends BindingBase with Scheduler, Renderer {
+   DemoBinding({ RenderBox root }) {
+    renderView.child = root;
+    ui.window.onPopRoute = handlePopRoute;
+    ui.window.onPointerPacket = handlePointerPacket;
+  }
 
-void handlePointerPacket(ByteData serializedPacket) {
-  bindings.Message message = new bindings.Message(
-    serializedPacket,
-    <core.MojoHandle>[],
-    serializedPacket.lengthInBytes,
-    0
-  );
-  PointerPacket packet = PointerPacket.deserialize(message);
+  void handlePopRoute() {
+    activity.finishCurrentActivity();
+  }
 
-  for (Pointer pointer in packet.pointers) {
-    if (pointer.type == PointerType.MOVE)
-      image.growth = math.max(0.0, image.growth + pointer.x - touches[pointer.pointer].x);
-    touches[pointer.pointer] = new Touch(pointer.x, pointer.y);
+  final Map<int, Touch> touches = <int, Touch>{};
+
+  void handlePointerPacket(ByteData serializedPacket) {
```

Comment:
The code here is pretty much unique to this demo. It used to just override the binding with this behaviour (without the binding knowing, it was a bit dodgy), in this patch it just has its own binding that does the behaviour it wants (and doesn't mix in the behaviour it doesn't want).